src/json-editor.component.html

```diff
@@ -9,7 +9,7 @@
   </div>
   <div id="editor-middle-column" [ngClass]="{widerColumn6: isPreviewerHidden}" class="col-md-6 middle" [shortcuts]="shortcuts">
     <table id="editor" class="table">
-      <div class="field-wrapper" *ngFor="let key of keys | addAlwaysShowFields:schema | filterAndSortBySchema:schema; trackBy:trackByElement">
+      <div class="field-wrapper" *ngFor="let key of keys | addAlwaysShowFields:schema | filterAndSortBySchema:schema; let last=last; trackBy:trackByElement" [ngClass]="{last:last}">
```

Comment:
So you tried with CSS pseudo selector to apply styling to the last element instead?